django/db/models/sql/query.py

```diff
@@ -2035,16 +2035,14 @@ def __init__(self, connector, num_children, negated):
         self.num_children = num_children
         # Maps of table alias to how many times it is seen as required for
         # inner and/or outer joins.
```

Comment:
I think "outer" should be kept since `votes` is still used in `update_join_types` to promote joins to `OUTER` or demote them to `INNER`.